src/util/kube_utils.js

```diff
@@ -69,7 +67,7 @@ function get_resource(resource_type, resource_name) {
 }
 
 function patch_resource(resource_type, resource_name, patch) {
-    return exec_kubectl(`patch ${resource_type} ${resource_name} -p='${JSON.stringify(patch)}'`, 'json');
+    return exec_kubectl(`patch ${resource_type} ${resource_name} --type merge -p='${JSON.stringify(patch)}'`, 'json');
```

Comment:
I thought that we decided to use kube_store, why did you introduce a change here. It may break old code that uses this util file. I would like us  to freeze this code until we move all of its call sites into using KubeStore instead.

As a note, all the other changes in this file are layout only changes. You can revert the whole file

Reply:
will revert